ClientApp/viking-vault/src/Login.tsx

```diff
@@ -187,7 +187,7 @@ class LoginForm extends React.Component<any, ILoginFormState> {
           {this.state.errorLabel !== "" ? <div className="alert alert-danger"> {this.state.errorLabel} </div> : null }
         </div>
         <FooterForm class="footer-login"></FooterForm>
```

Comment:
If FooterForm does not receive children, consider using self closing HTML tags.
`<FooterForm class="footer-login" />`